_data/harnesses/multi-joins/ksql.yml

```diff
@@ -0,0 +1,132 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/multi-joins/ksql/markup/dev/init.adoc
+
+        - change_directory: multi-joins
+          action: execute
+          file: tutorial-steps/dev/make-dirs.sh
+          render:
+            file: tutorials/multi-joins/ksql/markup/dev/make-dirs.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/multi-joins/ksql/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/multi-joins/ksql/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Write the program interactively using the CLI
```

Comment:
This section is quite big.  It would help to split this up maybe into two separate parts:

1. Creating the streams/tables and inserting values
2. Creating the multi-way joined stream uses common SQL join syntax.

Reply:
+1 on breaking up this section

Reply:
I'm not sure how to break up into sections.   What should be the values of a new "title" for content.action, docker_bootup_file_render, etc...?
```
      - title: Create the multi-way join
        content:
        - action: docker_ksql_cli_session
          container: ksqldb-cli
          docker_bootup_file: tutorial-steps/dev/start-cli.sh
          column_width: 17
          render:
            file: tutorials/multi-joins/ksql/markup/dev/start-cli.adoc
          stdin:
```

Reply:
I think I may have figured out my questions, please review the latest @bbejeck & @ybyzek